javascript/javascript1/week3/smart-ease.js

```diff
@@ -0,0 +1,60 @@
+const notes = [];
+
+const notepad = [
+  {
+    content: "Pick up groceries",
+    id: 1,
+  },
+
+  {
+    content: "Do laundry",
+    id: 2,
+  },
+];
+
+function saveNote(content, id) {
+  // write some code her
+  for (let note of notepad) {
+    notes.push(note);
+  }
+  return notes;
```

Comment:
You have just to push a note with the passed parameters `notes.push({ content, id });`